src/bezier.js

```diff
@@ -148,7 +148,7 @@ function cubicBezierAtTime(t, p1x, p1y, p2x, p2y, duration) {
  *  @return {Function}
  *  @private
  */
-const getCubicBezierTransition = (x1, y1, x2, y2) => pos =>
+export const getCubicBezierTransition = (x1, y1, x2, y2) => pos =>
```

Comment:
I can't comment directly on the line above since it wasn't changed, but since this is now exported we should remove the `@private` on line 149.

Reply:
I agree, I forgot to remove that!